binding.gyp

```diff
@@ -9,14 +9,25 @@
       'src/sharp.cc'
     ],
     'variables': {
-      'PKG_CONFIG_PATH': '<!(which brew >/dev/null 2>&1 && eval $(brew --env) && echo $PKG_CONFIG_LIBDIR || true):$PKG_CONFIG_PATH:/usr/local/lib/pkgconfig:/usr/lib/pkgconfig'
+      'PKG_CONFIG_PATH': '<!(which brew >/dev/null 2>&1 && eval $(brew --env) && echo $PKG_CONFIG_LIBDIR || true):$PKG_CONFIG_PATH:/usr/local/lib/pkgconfig:/usr/lib/pkgconfig',
+      'HAS_OPENSLIDE': '<!(./has_openslide.sh)'
     },
-    'libraries': [
-      '<!(PKG_CONFIG_PATH="<(PKG_CONFIG_PATH)" pkg-config --libs vips)'
-    ],
-    'include_dirs': [
-      '<!(PKG_CONFIG_PATH="<(PKG_CONFIG_PATH)" pkg-config --cflags vips glib-2.0)',
-      '<!(node -e "require(\'nan\')")'
+    'conditions': [
```

Comment:
Is this conditionality required? `pkg-config --cflags vips` already includes the openslide dependency (when present).